From 34076acbec085979e581c75ef2bf904a1aab5879 Mon Sep 17 00:00:00 2001
From: Nishanth Menon <nm@ti.com>
Date: Tue, 10 Dec 2013 23:01:15 -0600
Subject: [PATCH 0716/1115] ARM: AM43xx: remove wrong init_late from machine
 descriptor

commit e455c636ddfc760eea84e736e0b974d3d6328621 (ARM: mach-omap2:
Introduce late_init function for AM43x) introduced .init_late =
am43xx_init_late

However, commit eea497ee2895101f0eeac36168deaba1c1047ee9 (ARM:
OMAP2+: Export SoC data to userspace) introduces .init_late =
am33xx_init_late, again!

currently, we have DT_MACHINE_START(AM43_DT, "Generic AM43 (Flattened
Device Tree)") with two init_late entries! remove the wrong one.

Reported-by: Dave Gerlach <d-gerlach@ti.com>
Signed-off-by: Nishanth Menon <nm@ti.com>
(cherry picked from commit 78ac624498264ea663b20693bad52ac9eee819c0)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm/mach-omap2/board-generic.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-omap2/board-generic.c b/arch/arm/mach-omap2/board-generic.c
index 84926c0..d41de61 100644
--- a/arch/arm/mach-omap2/board-generic.c
+++ b/arch/arm/mach-omap2/board-generic.c
@@ -230,7 +230,6 @@ DT_MACHINE_START(AM43_DT, "Generic AM43 (Flattened Device Tree)")
 	.init_late	= am43xx_init_late,
 	.init_irq	= omap_gic_of_init,
 	.init_machine	= omap_generic_init,
-	.init_late	= am33xx_init_late,
 	.init_time	= omap3_sync32k_timer_init,
 	.dt_compat	= am43_boards_compat,
 	.restart	= am43xx_restart,
-- 
1.7.5.4

